src/gcov.rs

```diff
@@ -64,50 +64,92 @@ pub fn run_gcov(
     Ok(())
 }
 
-fn is_recent_version(gcov_output: &str) -> bool {
+pub fn get_gcov_version() -> &'static Version {
+    lazy_static! {
+        static ref V: Version = {
+            let output = Command::new(&get_gcov())
+                .arg("--version")
+                .output()
+                .expect("Failed to execute `gcov`. `gcov` is required (it is part of GCC).");
+            assert!(output.status.success(), "`gcov` failed to execute.");
+            let output = String::from_utf8(output.stdout).unwrap();
+            parse_version(&output)
+        };
+    }
+    &V
+}
+
+pub fn get_gcov_output_ext() -> &'static str {
+    lazy_static! {
+        static ref E: &'static str = {
+            if get_gcov_version().major >= 10 {
+                ".gcov.json.gz"
+            } else {
+                ".gcov"
+            }
+        };
+    }
+    &E
+}
+
+fn parse_version(gcov_output: &str) -> Version {
+    let mut versions: Vec<_> = gcov_output
+        .split(|c| c == ' ' || c == '\n')
+        .filter_map(|value| Version::parse(value).ok())
+        .collect();
+    assert!(versions.len() >= 1, "no version found for `gcov`.");
+
+    versions.pop().unwrap()
+}
+
+pub fn check_gcov_version() -> bool {
```

Comment:
Where is this one used other than tests? I couldn't find it

Reply:
I've no idea, I let this code because it was there...

Reply:
Let's file an issue to remove it then